udata/search/__init__.py

```diff
@@ -121,7 +121,7 @@ def get_i18n_analyzer():
 i18n_analyzer = LocalProxy(lambda: get_i18n_analyzer())
 
 
-@celery.task
+@task(queue='high', routing_key='high.search')
```

Comment:
why not `route='high.search'`? To stay consistent.

Reply:
Just forgot to remove the `queue`. I added the `route` after as a shortcut.